sql-plugin/src/main/scala/org/apache/spark/sql/rapids/complexTypeExtractors.scala

```diff
@@ -210,4 +203,45 @@ case class GpuArrayContains(left: Expression, right: Expression)
     lhs.getBase.listContainsColumn(rhs.getBase)
 
   override def prettyName: String = "array_contains"
-}
\ No newline at end of file
+}
+
+object GetArrayItemUtil {
+  def evalColumnar(array: GpuColumnVector, ordinal: Scalar, dataType: DataType,
```

Comment:
Public API needs doc

Reply:
Done.

Reply:
nit: A standard doc would be better, including params, returns.